build.gradle

```diff
@@ -0,0 +1,93 @@
+import com.github.spotbugs.SpotBugsTask
+
+plugins {
+  id "com.github.hierynomus.license" version "0.14.0" apply false
+  id 'com.github.johnrengelman.shadow' version '2.0.1' apply false
+  id "com.google.protobuf" version "0.8.3" apply false
+  id "com.github.spotbugs" version "1.6.0" apply false
+}
+
+allprojects {
+  apply plugin: 'idea'
+  group = 'cz.seznam.euphoria'
+  version = '0.8-SNAPSHOT'
+}
+
+subprojects {
+
+  apply plugin: 'java'
+  apply plugin: 'com.github.hierynomus.license'
+  apply plugin: 'com.github.spotbugs'
+  apply from: "${rootProject.projectDir}/gradle/versions.gradle"
+
+  license {
+    include '**/*.java'
```

Comment:
Not sure about this ... first of all, the maven license plugin checked much more files, including shell scripts, XML files, and many more ... we shouldn't drop this functionality.
Second doubt is about the year, I think it should stay as it is - i.e. `2016-<currentYear>`.